lesson_06/conditionals/conditionals_app/src/main/java/com/codedifferently/lesson6/Lesson6.java

```diff
@@ -61,7 +89,11 @@ public static String convertGpaToLetterGrade(double gpa) {
    * @return The factorial of n.
    */
   public static int computeFactorial(int n) {
-    return 0;
+    int factorial = 1;
```

Comment:
Should throw an exception if `n < 0`, think you can make that happen?

Reply:
This should now throw an exception if n < 0.